rust/geoparquet/src/writer/options.rs

```diff
@@ -22,6 +22,9 @@ pub struct GeoParquetWriterOptions {
     /// Set the type of encoding to use for writing to GeoParquet.
     pub encoding: GeoParquetWriterEncoding,
 
+    /// Set the primary geometry column name.
+    pub primary_column: Option<String>,
```

Comment:
I think we want some validation that this column 

1. does exist in the data 
2. is spatial and has associated metadata (essentially that it exists in the GeoParquet metadata)